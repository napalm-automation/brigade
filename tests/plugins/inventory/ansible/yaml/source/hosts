all:
    children:
        servers:
            children:
                webservers:
                    hosts:
                        foo.example.com:
                        bar.example.com:
                dbservers:
                    hosts:
                        one.example.com:
                        two.example.com:
                            my_var: from_hostfile
                        three.example.com:
                            ansible_port: 5555
                            ansible_host: 192.0.2.50
            vars:
                some_server: foo.southeast.example.com
                halon_system_timeout: 30
                self_destruct_countdown: 60
                escape_pods: 2
        frontend:
            hosts:
                foo.example.com:
