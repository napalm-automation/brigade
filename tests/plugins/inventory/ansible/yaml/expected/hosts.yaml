bar.example.com:
  groups:
  - webservers
foo.example.com:
  groups:
  - frontend
  - webservers
one.example.com:
  groups:
  - dbservers
  my_var: from_one.example.com
three.example.com:
  brigade_host: 192.0.2.50
  brigade_ssh_port: 5555
  groups:
  - dbservers
two.example.com:
  groups:
  - dbservers
  my_var: from_hostfile
  whatever: asdasd
